narayana-release-process.sh

```diff
@@ -198,55 +158,37 @@ else
 fi
 
 rm -rf $PWD/localm2repo
-./build.sh clean install -Dmaven.repo.local=${PWD}/localm2repo -DskipTests -gs ~/.m2/settings.xml -Dorson.jar.location=$ORSON_PATH -Pcommunity
+./build.sh clean install -Dmaven.repo.local=${PWD}/localm2repo -DskipTests -Dorson.jar.location=$ORSON_PATH -Pcommunity
 if [[ $? != 0 ]]
 then
   echo 1>&2 Could not install narayana
   exit
 fi
+# It is important in the deploy step that if you are deploying to nexus you provide a reference to your settings file as the ./build.sh overrides the default settings file discovery of Maven. Please see https://github.com/jbosstm/narayana/wiki/Narayana-Release-Process for details of the settings.xml requirements
 ./build.sh clean deploy -Dmaven.repo.local=${PWD}/localm2repo -DskipTests -gs ~/.m2/settings.xml -Dorson.jar.location=$ORSON_PATH -Prelease,community -DskipNexusStagingDeployMojo=false
 if [[ $? != 0 ]]
 then
   echo 1>&2 Could not deploy narayana to nexus
   exit
 fi
 git archive -o ../../narayana-full-$CURRENT-src.zip $CURRENT
```

Comment:
Maybe we need to guide why this is done (for the manual step of updating the website)

Reply:
Oh, I see. I think that can be removed from this script. I think that step is now done automatically by github: https://github.com/jbosstm/narayana/tags
So we only link that from the website (e.g. https://www.narayana.io/downloads/)

Reply:
OK! Thank you!